CSharp/Common/SampleHelpers.cs

```diff
@@ -144,15 +149,11 @@ public static async Task<List<ResourceFile>> UploadResourcesAndCreateResourceFil
         /// <param name="cloudStorageAccount">The cloud storage account to upload the file to.</param>
         /// <param name="containerName">The name of the container to upload the blob to.</param>
```

Comment:
container != containerName

Reply:
Oops thought they were the same. Anyways missed this method header in my initial review